src/pages/AppList.svelte

```diff
@@ -26,16 +27,23 @@
   <input type="checkbox" bind:checked={showDeprecated} />
   Show deprecated applications
 </label>
+<label>
+  <input type="checkbox" bind:checked={showPrototype} />
+  Show prototype applications
+</label>
```

Comment:
I will leave the main review to @brizental, however just a quick point that I don't think this part is necessary -- we can just go ahead and always show prototype applications for now (I don't expect there to be that many of them vs. deprecated ones).